src/Kernel/ClassDescription.class.st

```diff
@@ -373,186 +371,149 @@ ClassDescription >> commentStamp: changeStamp [
+		self compile: code classified: protocol ]
 ]
 
 { #category : #copying }
-ClassDescription >> copyAll: selArray from: class [
+ClassDescription >> copyAll: selectors from: class [
 	"Install all the methods found in the method dictionary of the second
 	argument, class, as the receiver's methods. Classify the messages under
 	-As yet not classified-."
 
-	self copyAll: selArray
-		from: class
-		classified: nil
+	self copyAll: selectors from: class classified: nil
 ]
 
 { #category : #copying }
-ClassDescription >> copyAll: selArray from: class classified: cat [
+ClassDescription >> copyAll: selectors from: class classified: protocolName [
 	"Install all the methods found in the method dictionary of the second
 	argument, class, as the receiver's methods. Classify the messages under
-	the third argument, cat."
+	the third argument, protocolName."
 
-	selArray do: [:s |
-		(class includesLocalSelector: s) ifTrue: [
-			self copy: s
-				from: class
-				classified: cat ] ]
+	selectors do: [ :selector | (class includesLocalSelector: selector) ifTrue: [ self copy: selector from: class classified: protocolName ] ]
 ]
 
 { #category : #copying }
 ClassDescription >> copyAllCategoriesFrom: aClass [
-	"Specify that the categories of messages for the receiver include all of
+	"Specify that the protocole of messages for the receiver include all of
 	those found in the class, aClass. Install each of the messages found in
-	these categories into the method dictionary of the receiver, classified
-	under the appropriate categories."
+	these protocols into the method dictionary of the receiver, classified
+	under the appropriate protocol."
 
-	aClass organization categories do: [:cat | self copyCategory: cat from: aClass]
+	aClass organization categories do: [ :protocolName | self copyCategory: protocolName from: aClass ]
 ]
 
 { #category : #copying }
-ClassDescription >> copyCategory: cat from: class [
-	"Specify that one of the categories of messages for the receiver is cat, as
-	found in the class, class. Copy each message found in this category."
+ClassDescription >> copyCategory: protocolName from: class [
+	"Specify that one of the protocol of messages for the receiver is protocolName, as
+	found in the class, class. Copy each message found in this protocol."
 
-	self copyCategory: cat
-		from: class
-		classified: cat
+	self copyCategory: protocolName from: class classified: protocolName
 ]
 
 { #category : #copying }
-ClassDescription >> copyCategory: cat from: aClass classified: newCat [
-	"Specify that one of the categories of messages for the receiver is the
-	third argument, newCat. Copy each message found in the category cat in
-	class aClass into this new category."
+ClassDescription >> copyCategory: protocolName from: aClass classified: newProtocolName [
+	"Specify that one of the protocol of messages for the receiver is the
```

Comment:
'protocol of messages ' sounds weird.

Reply:
I don't understand the whole sentence in fact.

Reply:
I changed the comment